roles/backend_setup/tasks/fscreate.yml

```diff
@@ -34,4 +34,4 @@
      opts: "{{ fs_options }} {{ raid_options | default('') }}"
      force: "{{ gluster_infra_fs_force | default('no') }}"
   with_items: "{{ gluster_infra_thick_lvs }}"
-  when: gluster_infra_thick_lvs is defined
+  when: gluster_infra_thick_lvs is defined and (item.skipfs is not defined or not item.skipfs)
```

Comment:
The change looks good. But just wondering why do we skip filesystem creation?
I am assuming that this is done allow provision to create other filesystems if needed?

Reply:
This is indeed one use case. But the primary reason i've added the option to skip the fs creation is of you want to host a VDO device on a Thick LV, the Thick LV cannot have a file system on it, and should be left as a raw device.

Reply:
> This is indeed one use case. But the primary reason i've added the option to skip the fs creation is of you want to host a VDO device on a Thick LV, the Thick LV cannot have a file system on it, and should be left as a raw device.

Oh okay, understood.